docs/getting_started/install-on-mac-os-and-windows.md

```diff
@@ -20,7 +20,7 @@ If you want to use a web server, you need to install it as well:
 
 The instructions below assume that you are using Apache.
 
-??? "Windows"
+??? note "Windows"
```

Comment:
Is there a reason why we are giving up the accordeon here and keeping it below for other platforms?

Reply:
Below there is a choice of steps between the systems, and this is a part of procedure that only applies to Win. This is only fixing the box, as it had been misformatted before.